frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -197,6 +197,32 @@ export default class AccountsApi extends ExplorerApi {
     }
   }
 
+  async queryDbAccount(accountId: string): Promise<any> {
```

Comment:
Given that the only reason we added this new method is to check if the given account id has ever existed, let's implement the method `isAccountIndexed` and return `Promise<boolean>`